lms/djangoapps/courseware/tests/test_credit_requirements.py

```diff
@@ -3,6 +3,7 @@
 """
 
 import datetime
+import ddt
```

Comment:
This is a third-party package and doesn't belong in the same block as standard-library imports.